tools/generator/src/Generator+Main.swift

```diff
@@ -39,21 +40,24 @@ extension Generator {
         let workspaceOutputPath: Path
         let projectRootDirectory: Path
         let buildMode: BuildMode
+        let automaticSigningTeamId: String
     }
 
     static func parseArguments(_ arguments: [String]) throws -> Arguments {
-        guard CommandLine.arguments.count == 7 else {
+        guard CommandLine.arguments.count == 8 else {
             throw UsageError(message: """
 Usage: \(CommandLine.arguments[0]) <path/to/project.json> \
 <path/to/xccurrentversions.json> <path/to/bazel/integration/dir> \
 <path/to/output/project.xcodeproj> <workspace/relative/output/path> \
-(xcode|bazel)
+(xcode|bazel) (teamId)
```

Comment:
Every time I add to this, I wonder if _now_ is the time to bring in the ArgumentParser dependency 🤔.